ui-components/src/blocks/Card.stories.tsx

```diff
@@ -0,0 +1,239 @@
+        />
+        <Card.Section centered={true}>
+          <LinkButton href="#" unstyled className="m-0">
+            Skip this and show me listings
+          </LinkButton>
+        </Card.Section>
+      </Card.Footer>
+    </Card>
+  </div>
+)
+
+export const DetroitStyle = () => {
+  const cssVarsOverride = `
+    .card-overrides {
+      --bloom-font-sans: Montserrat;
+      --bloom-font-alt-sans: var(--bloom-font-sans);
+      --bloom-color-primary: rgb(41,126,115);
+      --bloom-color-primary-dark: rgb(0,68,69);
+
+      --primary-appearance-hover-background-color: white;
+      --primary-appearance-hover-label-color: var(--bloom-color-primary-dark);
+
+      --outlined-appearance-hover-background-color: var(--bloom-color-primary);
+      --outlined-appearance-hover-border-color: var(--bloom-color-primary);
+
+      font-family: var(--bloom-font-sans);
+    }
+
+    .card-overrides h2 {
+      font-family: var(--bloom-font-sans);
+    }
+
+    .card-overrides .button {
+      --normal-rounded: 60px;
+      --normal-padding: 0.5rem 1rem;
+      --normal-font-size: var(--bloom-font-size-base);
+      --label-letter-spacing: normal;
+      --label-transform: none;
+    }
+  `
+
+  return (
+    <>
+      <link rel="preconnect" href="https://fonts.googleapis.com" />
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+      <link
+        href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap"
+        rel="stylesheet"
+      ></link>
+      <div className="card-overrides" style={{ maxWidth: "600px" }}>
+        <Card>
+          <Card.Header>
+            <HeadingGroup
+              headingPriority={2}
+              heading="Household Maximum Income"
+              subheading="To determine your eligibility for this property, choose your household size (include yourself in that calculation)."
+            />
+          </Card.Header>
+
+          <Card.Section className="markdown">
+            <div className="field">
+              <input type="radio" name="household-size" id="one-two" />
+              <label htmlFor="one-two">1-2 People</label>
+            </div>
+            <div className="field">
+              <input type="radio" name="household-size" id="three-six" />
+              <label htmlFor="three-six">3-6 People</label>
+            </div>
+          </Card.Section>
+
+          <Card.Footer>
+            <ButtonGroup
+              columns={[
+                <Button>Previous</Button>,
+                <Button styleType={AppearanceStyleType.primary}>Next</Button>,
+              ]}
+            />
+            <Card.Section centered={true}>
+              <LinkButton href="#" unstyled className="m-0">
+                Skip this and show me listings
+              </LinkButton>
+            </Card.Section>
+          </Card.Footer>
+        </Card>
+        <style>{cssVarsOverride}</style>
+      </div>
+    </>
+  )
+}
+
+export const CustomWackyCard = () => {
```

Comment:
Open to others' thoughts as well, but I'd prefer to remove this story, or to edit is so that it isn't quite as jarring visually (which I get is the point ;)). We already have the Detroit story to showcase visual styling.

Reply:
No worries, I'll remove that.